buildSrc/src/main/groovy/MavenToolchain.groovy

```diff
@@ -26,11 +26,11 @@ class MavenToolchain {
 
   private static def exe = OperatingSystem.current().isWindows() ? '.exe' : ''
 
-  static def javaHome = { v -> 
+  static def javaHome = { v ->
```

Comment:
sorry about the unnecessary whites spaces.  I will try to avoid this happening in the future.  Louis, I see you don't mind the one above.  Would you like me to update the one below?